docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,171 @@
+[Back to README](../README.md)
+
+## Contributing
+
+MyLA is fully open source and being actively developed by
+[Information and Technology Services, University of Michigan](https://its.umich.edu/) and
+[Centre for Teaching, Learning and Technology, University of British Columbia](https://ctlt.ubc.ca/).
+The backend is written in Python using the Django framework,
+and the frontend is written in JavaScript using React.
+We welcome any kind of contribution, from code to improvements in documentation and suggestions for future development.
+
+### Technology Overview
+
+#### Frontend
+
+The frontend of MyLA is written in JavaScript using [React](https://reactjs.org/), with
+[Material-UI](https://material-ui.com/) as the component library and [D3](https://d3js.org/) for the data visualizations.
+
+The code for the frontend is located under `/assets/src/`.
+MyLA uses functional components instead of class-based components and
+uses [React hooks](https://reactjs.org/docs/hooks-intro.html) for state and side-effects.
+[Jest](https://jestjs.io/) is used as the testing framework; see [Testing](testing.md) for more information.
+
+[Standard](https://standardjs.com/) is the JavaScript linter.
+Please ensure any new code is Standard-complaint by running the linter manually. It is installed as a dev dependency.
+There are also [text editor plugins](https://standardjs.com/index.html#are-there-text-editor-plugins) available.
+
+```
+npm install
```

Comment:
I've run `npm install` at myla root folder. However, I got `zsh: command not found: standard` error for the following `standard assets/src` code.

Reply:
I'll check.

Reply:
The command worked for me cause I had `standard` installed globally. Looks like you need to use `npx standard` if it's just in installed for the project: https://standardjs.com/index.html#usage

See https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1398/commits/74704d4d1970042fc49301591cf502e00018441a